platform/darwin/src/MGLCustomVectorSource.h

```diff
@@ -0,0 +1,65 @@
+#import "MGLSource.h"
+#import "MGLGeoJSONSource.h"
+#import "MGLGeoJSONSourceBase.h"
+#import "MGLGeometry.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol MGLFeature;
+
+/**
+ Data source for `MGLCustomVectorSource`.
+ */
+@protocol MGLCustomVectorSourceDataSource <NSObject>
+
+/** 
+ Fetch data for a tile
+ @param zoom
+ @param y
+ @param x
```

Comment:
When filling in the documentation, make sure to mention that this method is always called on the operation queue in the `requestQueue` property.